tests/test_config.py

```diff
@@ -7,20 +7,25 @@
 PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I changed nothing in this file except formatting (autoformat of PyCharm).
If you don't want that just let me know.